t3.py

```diff
@@ -0,0 +1,79 @@
+# Example of a very bad, insecure, and buggy Python script
+
+# 1. Hardcoded sensitive credentials and secrets
+DATABASE_PASSWORD = "superSecret123"  # Hardcoded password (BAD PRACTICE)
+API_KEY = "abc123apikeysupersecret"  # Hardcoded API key (BAD PRACTICE)
+PRIVATE_KEY = """
+-----BEGIN PRIVATE KEY-----
+MIICXQIBAAKBgQCv+bigbadexamplefakeprivatekeywhichislongbutwrong
+Y6lOCIxJkLDxJnJ7lf6F8bigerrorpronecodeexposedinplaintextsuperbad
+-----END PRIVATE KEY-----
+"""  # Exposing private key (HUGE SECURITY RISK)
```

Comment:
Hardcoded sensitive credentials and secrets pose a significant security risk. It's recommended to use environment variables or secure vaults for storing sensitive information.

**Label**: security

**Code suggestion**

```python
import os

DATABASE_PASSWORD = os.getenv('DATABASE_PASSWORD')
API_KEY = os.getenv('API_KEY')
PRIVATE_KEY = os.getenv('PRIVATE_KEY')
```